server/controller/userController.js

```diff
@@ -0,0 +1,116 @@
+import jwt from 'jsonwebtoken';
+import pg from 'pg';
+
+import config from './../config/config';
+import user from './../seed/user';
+
+const { connect } = user;
+
+const client = new pg.Client(config);
+
+require('dotenv').config();
+
+connect();
+
+/**
+ * it is a class that control all event method
+ */
+class UserController {
+  /**
+ * it create a new user
+ * @param {string} req
+ * @param {string} res
+ * @return {object} an object
+ */
+  static async signUp(req, res) {
+    try {
+      const {
+        firstName, lastName, phoneNumber, email, password
+      } = req.body;
+
+      const checkEmail = `
```

Comment:
This is good work but one question...is your phone number not unique too? Why not checking if it existence too